src/forbidden/ad-test-signatures.txt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+@defaultMessage use format with Locale
+java.lang.String#format(java.lang.String,java.lang.Object[])
```

Comment:
Will this fail `./gradlew build` if found format not using Locale?

Reply:
It will fail as of now. I can add `ignoreFailures = true` back if we want to just have warning show up.

Reply:
It's good to fail. Just ask to confirm.